src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -57,7 +57,7 @@ public PersonCard(Person person, int displayedIndex) {
         address.setText(person.getAddress().value);
         email.setText(person.getEmail().value);
         payment.setText(person.getPayment().toString());
-        attendance.setText(person.getAttendance().toString());
+        attendance.setText("Attended on: " + person.getAttendance().toDisplayString());
```

Comment:
Would it better if the "Attended on: " is included in the toDisplayString() method? 
So that over here could have setText(person.getAttendance().toDisplayString()) straight away

Reply:
Good idea